model/map-jpa/src/main/resources/META-INF/single-use-objects/jpa-single-use-objects-changelog-1.xml

```diff
@@ -47,9 +47,6 @@ limitations under the License.
         <createIndex tableName="kc_single_use_obj" indexName="single_use_obj_expiration">
             <column name="expiration"/>
         </createIndex>
-        <ext:createJsonIndex tableName="kc_single_use_obj" indexName="kc_single_use_obj_nonce">
-            <ext:column jsonColumn="metadata" jsonProperty="fActionVerificationNonce"/>
-        </ext:createJsonIndex>
```

Comment:
I think this drop of the index has to go to different `changeSet`. Otherwise it would fail with validation when checking the checksums when running in no-downtime mode.

Reply:
Thanks. I fixed it. It should work as expected:
```
keycloak=# \d kc_single_use_obj
...
Indexes:
    "kc_single_use_obj_pkey" PRIMARY KEY, btree (id)
    "single_use_obj_entityVersion" btree (entityversion)
    "single_use_obj_expiration" btree (expiration)
    "single_use_obj_objectKey" btree (objectkey)
Referenced by:
    ...
```
There is no longer the following index: `"kc_single_use_obj_nonce" gin ((metadata -> 'fActionVerificationNonce'::text) jsonb_path_ops)`

Reply:
Thank you. still it should be reverted to pass the validation.